src/test/kotlin/com/esosa/f5pi_backend/controllers/implementations/AuthControllerTest.kt

```diff
@@ -52,4 +54,60 @@ class AuthControllerTest : BaseIntegrationTest() {
 
         assertEquals(userCountBefore?.plus(1), userCountAfter)
     }
+
+    @Test
+    fun `should not check token with SQL injection payload`() {
+        RegisterRequest(
+            "DELETE FROM USER WHERE 1 = 1",
+            "DELETE FROM GAME WHERE 1 = 2",
+            "DELETE FROM FIELD WHERE 1 = 1",
+            "test@gmail.com"
+        ).also {
+            mockMvc.perform(
+                post("/auth/register")
+                    .contentType(MediaType.APPLICATION_JSON)
+                    .content(toJson(it))
+            ).andExpect(status().isCreated)
+        }.also {
+            mockMvc.perform(
+                post("/auth/login")
+                    .contentType(MediaType.APPLICATION_JSON)
+                    .content(toJson(LoginRequest(it.username, it.password)))
+            ).andExpect(status().isOk)
+        }.also {
+            mockMvc.perform(
+                post("/auth/check-token")
+                    .contentType(MediaType.APPLICATION_JSON)
+                    .content(toJson(CheckTokenRequest("DELETE FROM USER WHERE 1 = 1")))
+            ).andExpect(status().isUnauthorized)
+        }
+    }
+
+    @Test
+    fun `should not refresh token with SQL injection payload`() {
+        RegisterRequest(
+            "DELETE FROM USER WHERE 1 = 1",
+            "DELETE FROM GAME WHERE 1 = 2",
```

Comment:
Si la intención es probar la SQL Injection esto jamás daría true jajaja

```suggestion
            "DELETE FROM GAME WHERE 1 = 1",
```